### DISCUSSION TITLE
how to start with sudo or under the root ?

### DISCUSSION THREAD

**Discussion by banrieen** [iongion/container-desktop#8] [Q&A]
when i execute the cmd:
sudo  podman-desktop-companion 
showns:

Running as root without --no-sandbox is not support , and a chrome bug:
Issue 638180: Exit instead of crashing when running as root without --no-sandbox.

**Comment by iongion** [comment]
Hi @banrieen - podman itself allows rootless containers due to all benefits this brings, https://developers.redhat.com/blog/2020/09/25/rootless-containers-with-podman-the-basics

Is there a particular reason you try to run it as root user ?
